apps/users-service/src/cms/index.ts

```diff
@@ -1,11 +1,15 @@
 import Users from './users';
+import Romps from './romps';
```

Comment:
The team's outputs overall are called the ROMP, so I think for the individual 'posts' we should find a different terminology. Maybe `ResearchOutput(s)` or just `Output(s)`? @fampinheiro

Reply:
I've been referring to them as `RompItems`, ReseachOutputs` is an option. Let me raise this to see the terminology product and design is using.